drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -18,6 +18,7 @@ class HmisSchema::Enrollment < Types::BaseObject
     field :services, HmisSchema::Service.page_type, null: false
     field :household, HmisSchema::Household, null: false
     field :client, HmisSchema::Client, null: false
+    field :relationship_to_ho_h, HmisSchema::Enums::RelationshipToHoH, null: false
```

Comment:
I don't think this makes sense as a field on Enrollment. It already exists on the household client, so this would mean we're storing it in two places.

Reply:
nvm ignore me, I forgot `RelationshipToHoH` is literally a column on the Enrollment table (not a separate household table) so I guess this makes sense

Reply:
Yeah, without this we actually don't really know what the RelationshipToHoH of this particular enrollment with this particular client without fetching full household information.